src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum BgMizuMovebgType {
+    /* 0 */ MIZUMOVEBG_TYPE_SMALL_BLOCK,
+    /* 1 */ MIZUMOVEBG_TYPE_MEDIUM_BLOCK,
+    /* 2 */ MIZUMOVEBG_TYPE_LARGE_BLOCK,
+    /* 3 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_1,
+    /* 4 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_2,
+    /* 5 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_3,
+    /* 6 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_4,
+    /* 7 */ MIZUMOVEBG_TYPE_SMALL_BLOCK_HOOKSHOT_TARGET,
+    /* 8 */ MIZUMOVEBG_TYPE_MAX
```

Comment:
Couple thoughts. First, I think I prefer calling them platforms over blocks. Second, everything except type 7 should be named based on their location. Types 0-2 are specifically modeled to match the locations they are spawned in, so I'd also update the names in the object file as well. Third, the hookshot platform is distinct enough to just be called as is

```suggestion
    /* 0 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_WEST
    /* 1 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_OUTSIDE_CENTER_PILLAR
    /* 2 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_INSIDE_CENTER_PILLAR
    /* 3 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_1, // is this the boss room statue?
    /* 4 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_2,
    /* 5 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_3,
    /* 6 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_4,
    /* 7 */ MIZUMOVEBG_TYPE_HOOKSHOT_PLATFORM
    /* 8 */ MIZUMOVEBG_TYPE_MAX
```

Reply:
type 3 is the boss room statue, how about ``MIZUMOVEBG_TYPE_DRAGON_STATUE_BOSS_ROOM``?

``MIZUMOVEBG_TYPE_DRAGON_STATUE_3`` is used on MQ but statue 2 and 3 aren't and they do the exact same thing as statue 3 btw

Reply:
sure